readthedocs/templates/projects/project_traffic_analytics.html

```diff
@@ -48,6 +48,27 @@ <h2>{% trans "Daily pageview totals" %}</h2>
     <button type="submit" name="download" value="true">{% trans "Download all data" %}</button>
   </form>
 
+  <h3>{% trans "Broken links"%}</h3>
```

Comment:
```suggestion
  <h3>{% trans "Not Found (404) pages"%}</h3>
  
  <p>
  {% blocktrans %}
  The way that we could 404 pages is on our backend hosting. It's possible the number of times each page is viewed is undercounted because of caching in the browser or in a CDN.
  {% endblocktrans %}
  </p>
```